src/test/java/org/jenkinsci/plugins/workflow/cps/steps/ingroovy/StepInGroovyTest.java

```diff
@@ -0,0 +1,40 @@
+package org.jenkinsci.plugins.workflow.cps.steps.ingroovy;
+
+import org.jenkinsci.plugins.workflow.cps.CpsFlowDefinition;
+import org.jenkinsci.plugins.workflow.job.WorkflowJob;
+import org.jenkinsci.plugins.workflow.job.WorkflowRun;
+import org.junit.Rule;
+import org.junit.Test;
+import org.jvnet.hudson.test.JenkinsRule;
+
+import java.net.MalformedURLException;
+import java.net.URL;
+
+/**
+ * @author Kohsuke Kawaguchi
+ */
+public class StepInGroovyTest {
+    @Rule
+    public JenkinsRule j = new JenkinsRule();
+
+    static {
+        try {
+            // fake content root at src/test/resources/stepsInGroovy
+            URL u = StepInGroovyTest.class.getResource("/stepsInGroovy/META-INF/index");
+            GroovyCompiler.additionalContentRoots.add(new URL(u,"../"));
```

Comment:
Or you can use `@WithPlugin` for a more realistic test (at the expense of having test sources inside a ZIP).

If my idea about using the plain resource path is feasible, then this would not be necessary.